client/test/app/hearings/components/scheduleHearing/TimeSlot.test.js

```diff
@@ -193,6 +193,23 @@ describe('TimeSlot', () => {
 
         });
 
+        it('moves following slots when there is a lunch break', () => {
+          const beginsAt = moment('2021-04-21T08:30:00-04:00').tz('America/New_York');
+          const lunchBreak = TIMEZONES_WITH_LUNCHBREAK.includes(ro.timezone);
+          const { timeSlots } = setup({ lunchBreak, beginsAt, roTimezone: ro.timezone });
+
+          expect(timeSlots[0].time.isSame(beginsAt)).toEqual(true);
+
+          const [breakHour, breakMinute] = ['12', '30'];
+          const lunchBreakMoment = beginsAt.clone().tz(ro.timezone).
+            set({ hour: breakHour, minute: breakMinute });
+          const firstSlotAfterLunchBreak = timeSlots.find((item) => item.time.isSameOrAfter(lunchBreakMoment));
+
+          if (lunchBreak) {
+            expect(firstSlotAfterLunchBreak.time.isSame(lunchBreakMoment.add(30, 'minutes'))).toEqual(true);
+          }
```

Comment:
```suggestion
          expect(firstSlotAfterLunchBreak.time.isSame(lunchBreakMoment.add(30, 'minutes'))).toEqual(lunchBreak);
```
Changing the expectation like this will test the non-lunch break-having time zones as well.